recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/gles-user-module_1.10.bbappend

```diff
@@ -0,0 +1,5 @@
+require inc/xt_shared_env.inc
+
+EXTRA_OEMAKE += "PVRSRV_VZ_NUM_OSID=${XT_PVR_NUM_OSID}"
```

Comment:
IMO, this can go into the inc file as well

Reply:
Done